packages/app/src/app/overmind/namespaces/files/internalActions.ts

```diff
@@ -93,8 +93,6 @@ export const uploadFiles: AsyncAction<
             /\.haml$/.test(filePath) ||
             /\.pug$/.test(filePath) ||
             /\.svg$/.test(filePath) ||
-            /\.md$/.test(filePath) ||
-            /\.svelte$/.test(filePath) ||
```

Comment:
I think this one should be added again :D

Reply:
Added :)